src/setuptools_pyproject_migration/__init__.py

```diff
@@ -278,8 +278,13 @@ def _generate(self) -> Pyproject:
             pyproject["project"]["description"] = description
 
         if dist.get_long_description() not in (None, "UNKNOWN"):
-            long_description_source: str = dist.command_options["metadata"]["long_description"][1]
+            long_description_source: str = dist.get_long_description()
+
+            if "metadata" in dist.command_options:
+                long_description_source = dist.command_options["metadata"]["long_description"][1]
```

Comment:
Personally I'd write it like this, mostly because it looks cleaner, but this also avoids one extra call to `get_long_description()` under certain circumstances:

```suggestion
            long_description_source: str
            if "metadata" in dist.command_options:
                long_description_source = dist.command_options["metadata"]["long_description"][1]
            else:
                long_description_source = dist.get_long_description()
```

Up to you though, I'd call this one an opinion.

Reply:
Ahh okay, I wasn't sure how to declare the variable type without assigning it a value.  Before, it moaned at me because I was declaring the type in two places.

Reply:
Yeah at some point in the past I ran into the same problem and had to look up how to do it. It's not the most obvious thing :sweat_smile: